authority-portal-frontend/src/app/pages/authority-organization-list-page/authority-organization-list-page/authority-organization-list-page.component.html

```diff
@@ -33,19 +33,25 @@
                 <th
                   class="py-3.5 px-4 text-sm font-normal text-left text-gray-700"
                   scope="col">
-                  No. of Users
+                  Users
                 </th>
 
                 <th
                   class="py-3.5 px-4 text-sm font-normal text-left text-gray-700"
                   scope="col">
-                  No. of Connectors
+                  Connectors
                 </th>
 
                 <th
                   class="py-3.5 px-4 text-sm font-normal text-left text-gray-700"
                   scope="col">
-                  No. of Data Offers
+                  Avail. Data Offers
```

Comment:
Data Offers (Live)

Reply:
I don't like this either but MDS wants it being called "Available" in their issue. I don't know how important that is so I opted for that. Maybe "Data Offers (Available)" idk